presto-docs/src/main/sphinx/functions/geospatial.rst

```diff
@@ -187,3 +187,46 @@ Accessors
 .. function:: ST_NumInteriorRing(Geometry) -> bigint
 
     Returns the cardinality of the collection of interior rings of a polygon.
+
+=====================
+Bing Tiles
+=====================
+
```

Comment:
In this doc all the functions above this section have descriptions starting with `Returns` while all the functions in this section have descriptions starting with `Given`. It would be nice to make that consistent.